composer.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "eppo/php-sdk",
   "description": "Eppo PHP SDK",
+  "version": "1.3.1",
```

Comment:
This was originally for testing. Not sure what the convention is here for version bumping, though I expect we will bump to v3 with the ufc migration.

Reply:
Dropping support for 7 could be a breaking change but since we will release a major version soon this makes sense to me; we can also leave as-is and I can give notice to our PHP customers about the upcoming change.